batch/scripts/report_status.sh

```diff
@@ -46,7 +45,7 @@ if [ $EXIT_CODE -eq 0 ] && [ $GREP_EXIT_CODE -ne 0 ]; then
 else
     STATUS="failed"
     MESSAGE="Command [ $ESC_COMMAND ] encountered errors"
-    DETAIL="$ESC_OUTPUT"
+    DETAIL=${ESC_OUTPUT:(-1000)} # crop output length to be able to send using CURL
```

Comment:
This still doesn't work. Detail field is still empty. Will create a seperate ticket for this.